tests/geo/test_surface_rzfourier.py

```diff
@@ -771,6 +771,24 @@ def test_fourier_transform_scalar(self):
         # Check that the result is the same as the original field:
         np.testing.assert_allclose(field/2*np.pi**2, field2)
 
+    def test_copy_method(self):
```

Comment:
Some lines of copy are not being tested (see codecov)

Reply:
Should be in the latest push, with a logic fix!